packages/data-stores/src/plans/hooks/use-intro-offers.ts

```diff
@@ -3,7 +3,7 @@ import usePlans from '../queries/use-plans';
 import useSitePlans from '../queries/use-site-plans';
 import type { PlanIntroductoryOffer } from '../types';
 
-interface IntroOffersIndex {
+export interface IntroOffersIndex {
```

Comment:
`Non-blocking:`

I'm not seeing this interface used elsewhere in this PR, so just hoping for a quick confirmation. I'm guessing that we export this `IntroOffersIndex` interface to eventually consolidate definitions with the `useIntroOffersForWooExpress` hook?

Reply:
I think it's just leftover code. now that you mentioned though, not a bad idea to consolidate these. maybe in a follow-up :-)